Google.PowerShell/Compute/GceMetadataCmdlet.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.PowerShell.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Net;
+
+namespace Google.PowerShell.ComputeEngine
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Gets the current metadata from the metadata server.
+    /// </para>
+    /// <para type="description">
+    /// Gets the current metadata from the metadata server. Get-GceMetadata can only be called from a Google
+    /// Compute Engine VM instance. Calls from any other machine will fail.
+    /// </para>
+    /// <example>
+    /// <code>PS C:\> $allMetadata = Get-GceMetadata | ConvertFrom-Json</code>
+    /// <para>Gets all of the metadata and converts it from the given json to a PSCustomObject.</para>
+    /// </example>
+    /// <example>
+    /// <code>PS C:\> $hostName = Get-GceMetadata -Path "instance/hostname" -NotRecursive</code>
+    /// <para>Gets the hostname of the instance.</para>
```

Comment:
Could you provide another example for getting a single value?

e.g. if I put a password into the metadata server, how would I get it?  `-Path "authtoken"`